src/renderers/dom/shared/ReactDOMComponent.js

```diff
@@ -210,6 +211,7 @@ function assertValidProps(component, props) {
 }
 
 function enqueuePutListener(inst, registrationName, listener, transaction) {
+  if (transaction instanceof ReactServerRenderingTransaction) return;
```

Comment:
Can you make this this a block and across multiple lines? Lint is only warning about it so not failing Travis but you can see it locally.